.github/workflows/Tests.yml

```diff
@@ -53,6 +53,18 @@ jobs:
 
     steps:
 
+      - name: Installing Klayout
+        run: |
+          sudo apt update -qq -y
+          wget https://www.klayout.org/downloads/Ubuntu-22/klayout_0.28.2-1_amd64.deb
```

Comment:
do we want to update to https://anaconda.org/LiteX-Hub/klayout to get klayout `HEAD`? do we want to stick to released point version?